src/Mesh/Grids.jl

```diff
@@ -304,6 +312,11 @@ function computegeometry(topology::AbstractTopology{dim}, D, ξ, ω, meshwarp,
       x3ξ3 = J * (ξ1x1 * ξ2x2 - ξ2x1 * ξ1x2)
       hypot(x1ξ3, x2ξ3, x3ξ3)
     end
+    map!(JH, J, ξ1x3, ξ2x3, ξ3x3
+        ) do J, ξ1x3, ξ2x3, ξ3x3
+      hypot(J*ξ1x3 ,J*ξ2x3, J*ξ3x3)
+    end
+    MHJH .= MH .* JH
```

Comment:
```julia
map!(MHJH, J, ξ3x1, ξ3x2, ξ3x3) do J, ξ3x1, ξ3x2, ξ3x3
  hypot(J*ξ3x1, J*ξ3x2, J*ξ3x3)
end
MHJH .= MH .* MHJH
```

Reply:
@jkozdon this is doesn't seem like it would change anything, is it just a suggestion for cleanup?

Reply:
Oh I see what you mean
Let me try that

Reply:
@jkozdon that was it. Thanks a lot, I really appreciate it. Going through with cleanup and will push for you to verify when it's done.